src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,246 @@
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed } from 'discord.js';
+import { LongLivingReactionCollector, LLRCData } from '../../lib/util/LongLivingReactionCollector';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Emojis } from '../../lib/util/constants';
+import { getColor } from '../../lib/util/util';
+
+enum ReactionEmoji {
+	HIGHER = '⬆',
+	LOWER = '⬇',
+	CANCEL = '❌', // TODO: Replace with <:redCross:637706251257511973>
+	OK = '✔️', // TODO: Replace with <:greenTick:637706251253317669>
```

Comment:
Seeing as this isn't const, there is an exported enum from `constants.ts` that has a RedCross and GreenTick enum members

Reply:
Actually, looking at it this can be made into a const enum and use those enum members.

Reply:
Rebuttal:
a) Enums don't allow their members to be members of other enums. I... what?
b) I use `Object.entries()` with the enum, which doesn't allow making it const